SillyBusinessInc/SillyBusinessGame
# Feat:#55 attack damage

## Purpose of this PR:
The tail and the slam all do damage
## How to test:
when you attack a enemy should take damage, you can drag a enemy prefab into a scene. Also the damages get logged and then you can  see if the damage is correct

### links: 
[Ticket](https://github.com/SillyBusinessInc/SillyBusinessGame/issues/55)
